webprojects/amo.md

```diff
@@ -0,0 +1,93 @@
+* Staging sever: the master is tagged and pushed to staging weekly on Tuesday at around 9am PST.
+* Production: the push usually starts at 9am PST every Thursday.
+
+## Advanced Configuration for testing strings from **addons-frontend**
+
+The above instruction is sufficient for basic localization testing. If you need to test strings related to the install buttons, you will need to enabled preferences to allow add-on installation to work on the development and stage servers.
+
+To fully configure Firefox to install add-ons on either dev or stage, you will
+need to do the following:
+
+* Use Firefox Nightly or Firefox Developer Edition.
+* A new profile is recommended since changing the dev certs will mark existing add-on installs as invalid (this change is not immediate).
+* Go to `about:config` and set `extensions.webapi.testing` as a Boolean to `true`.
+* To install an add-on, you will need to be able to install a signed add-on from dev or stage. Create the `xpinstall.signatures.dev-root` pref in `about:config` as a Boolean and set it to `true`.
+* For Disco Pane: Go to `about:config` and change the host in `extensions.webservice.discoverURL` to “https://discovery.addons-dev.allizom.org/“ for Dev server or “https://discovery.addons.allizom.org/“ for Staging server.
+* If you see warnings when trying to install a theme or theme previews doesn’t work, go to `about:preferences#security` and add an exception for dev or stage. You can do so by  clicking the `Exceptions` button after selecting Privacy & Security on left panel.
+* Restart the browser.
+
+## Additional advance testing
+
+To gain access to detailed test cases for all the features in various scenarios, you need to have LDAP credential as detailed in [this wiki page](https://wiki.mozilla.org/TestEngineering/Testrail)
+
+* [Sanity test for AMO](https://testrail.stage.mozaws.net/index.php?/suites/view/108&group_by=cases:section_id&group_order=asc)
+* [Add-ons manager/ Disco pane](https://testrail.stage.mozaws.net/index.php?/suites/view/98&group_by=cases:section_id&group_order=asc)
+* [Reviewer tools](https://testrail.stage.mozaws.net/index.php?/suites/view/100&group_by=cases:section_id&group_order=asc)
+* [AMO developer pages](https://testrail.stage.mozaws.net/index.php?/suites/view/99&group_by=cases:section_id&group_order=asc)
+* [Login/Register flows](https://testrail.stage.mozaws.net/index.php?/suites/view/95&group_by=cases:section_id&group_order=asc)
+
+## Debug locales (or i18n testing)
+
+There are two debug locales generated from the English keys. These can be useful to detect strings that are not marked for extraction.
+
+The debug locales are called “dbl“ (Debug left to right) and “dbr“ (Debug right to left). The tool used to build these is called “potools“, and it is both HTML and placeholder aware, to provide more accurate simulated localizations.
```

Comment:
No quotes to highlight.

Why would this information be useful for localizers?

Reply:
Dropped this one too.